pkgs/by-name/ro/rogcat/package.nix

```
@@ -0,0 +1,44 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  versionCheckHook,
+
+  pkg-config,
+  libudev-zero,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "rogcat";
+  version = "0.4.7";
+
+  src = fetchFromGitHub {
+    owner = "flxo";
+    repo = "rogcat";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-l2zfVt2vm5GTrYs6/0D3EesxxPWSmjf2tGS545766iA=";
+  };
+
+  cargoHash = "sha256-cDAS8mengFgBsq9nTiVAjt7pJhKjj7/F9x8IS6vP2ck=";
+
+  nativeBuildInputs = [
+    pkg-config
+    versionCheckHook
+  ];
+
+  buildInputs = [
+    libudev-zero
+  ];
+
+  versionCheckProgramArg = [ "--version" ];
```

```suggestion
  doInstallCheck = true;
  
  nativeInstallCheckInputs = [
    versionCheckHook
  ];

  versionCheckProgramArg = [ "--version" ];
```

versionCheckHook does not run currently. See ofborg